src/components/events/events.module.scss

```diff
@@ -74,6 +74,10 @@
   }
 }
 
+.linkBox > div > span > span {
```

Comment:
This change probably be removed if we change the Image element in EventList.tsx to:
```
<Image
                      src={event.field_image_url}
                      alt={event.field_image_alt}
                      layout='responsive'
                      objectFit='cover'
                      width={3}
                      height={2}
                    />
```